ngrinder-controller/src/main/java/org/ngrinder/user/controller/UserApiController.java

```diff
@@ -173,15 +176,13 @@ public void save(User user, @RequestBody User updatedUser) {
 			checkArgument(updatedUserInDb.getId().equals(updatedUser.getId()), "Illegal request to update user:%s",
 				updatedUser);
 		}
-		save(updatedUser);
+		save(user.getUserId(), updatedUser);
 	}
 
-	private User save(User user) {
-		if (StringUtils.isBlank(user.getPassword())) {
-			return userService.saveWithoutPasswordEncoding(user);
-		} else {
-			return userService.save(user);
-		}
+	private User save(String modifier, User user) {
```

Comment:
I don't understand.. why the first parameter name is modifier?

Reply:
I tried to indicate who had modified(save, update) the user information.